lib/screens/main_screen/overview_screen/widgets/my_health_section.dart

```diff
@@ -38,30 +38,62 @@ class MyHealthSection extends StatelessWidget {
           spacing: 20,
           runSpacing: 20,
           children: <Widget>[
-            Tile(
-              imageName: 'assets/Medikationsplan.png',
-              sectionName: AppLocalizations.of(context)!.myMedications,
-              routeName: '/my-medications-screen/my-medications',
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Flexible(
```

Comment:
Are these Flexible widgets necessary? The Tile widget already includes Flexible.